src/shared/utilities/childProcess.ts

```diff
@@ -96,15 +96,15 @@ export class ChildProcess {
                 params.onClose(code, signal)
             }
 
-            this.childProcess!.stdout.removeAllListeners()
-            this.childProcess!.stderr.removeAllListeners()
+            this.childProcess!.stdout?.removeAllListeners()
+            this.childProcess!.stderr?.removeAllListeners()
             this.childProcess!.removeAllListeners()
         })
     }
 
-    public kill(signal?: string): void {
+    public kill(): void {
         if (this.childProcess && !this.killed) {
-            this.childProcess.kill(signal)
+            this.childProcess.kill()
```

Comment:
Is SIGTERM the only signal we'll ever want to send?

Reply:
At the moment yes. The issue here was that it was taking a string but the type has been updated to be a typed `signal`. Since we do not use it, might as well delete it. If we ever need different signals we can add it back properly.